src/test/java/thrift/logic/commands/AddIncomeCommandIntegrationTest.java

```diff
@@ -56,4 +57,30 @@ public void undo_success() {
         expectedModel.deleteTransaction(transactionToDelete);
         assertEquals(expectedModel, model);
     }
+
+    @Test
+    public void redo_redoAddIncome_success() throws CommandException {
```

Comment:
Is the method name supposed to be `undo_redoAddIncome_success` because it does `undo` and `redo`?